pkg/features/kube_features.go

```diff
@@ -681,6 +681,12 @@ const (
 	//
 	// Enables the usage of different protocols in the same Service with type=LoadBalancer
 	MixedProtocolLBService featuregate.Feature = "MixedProtocolLBService"
+
+	// owner: @fromani
+	// alpha: v1.21
+	//
+	// Enable POD resources API to return allocatable compute resources
```

Comment:
This "compute resources" is a little bit misleading. Please consider just calling it "allocatable resources".

Reply:
ok, renaming to "allocatable resources"